### PR TITLE
New package: flite-2.0.0

### PR DESCRIPTION
**Pull request by Sir-Boops**
* The license although custom is very permissive [Link](http://www.festvox.org/flite/download.html)

### CODE REVIEW

File: srcpkgs/flite/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+license="Custom"
+homepage="http://www.festvox.org/flite/"
+distfiles="http://www.festvox.org/flite/packed/flite-${version%.*}/flite-${version}-release.tar.bz2"
+checksum=678c3860fd539402b5d1699b921239072af6acb4e72dc4720494112807cae411
+
+post_install() {
+	vlicense ${wrksrc}/COPYING
+}
+
+flite-devel_package() {
```
Comment (ebfe) [inline srcpkgs/flite/template]: short_desc and depends missing here.

File: srcpkgs/flite/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+makedepends="alsa-lib-devel"
+short_desc="A lightweight speech synthesis engine"
+maintainer="Sir_Boops <admin@boops.me>"
+license="Custom"
+homepage="http://www.festvox.org/flite/"
+distfiles="http://www.festvox.org/flite/packed/flite-${version%.*}/flite-${version}-release.tar.bz2"
+checksum=678c3860fd539402b5d1699b921239072af6acb4e72dc4720494112807cae411
+
+post_install() {
+	vlicense ${wrksrc}/COPYING
```
Comment (ebfe) [inline srcpkgs/flite/template]: drop ${wrksrc}

File: srcpkgs/flite/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+pkgname=flite
+version=2.0.0
+revision=1
+wrksrc="flite-${version}-release"
+build_style=gnu-configure
+configure_args="--enable-shared --with-audio=alsa"
+makedepends="alsa-lib-devel"
+short_desc="A lightweight speech synthesis engine"
+maintainer="Sir_Boops <admin@boops.me>"
+license="Custom"
```
Comment (ebfe) [inline srcpkgs/flite/template]:
```
$ git grep license.*custom | wc -l
35
$ git grep license.*Custom | wc -l
7
```

I vote for `custom`.

**Comment by Sir-Boops** [comment]
Updated the pr with the fixes

**Pull request closed without merging**